### DISCUSSION TITLE
investpy.economic_calendar

### DISCUSSION THREAD

**Discussion by varshasahasrabuddhe** [alvarobartt/investiny#61] [Q&A]
Hi, 
investpy.economic_calendar is failing with below error.
Getting exogs from investing.com, iteration 1
Expecting value: line 1 column 1 (char 0)

How can i use investiny  ?

**Comment by przemyslawbak** [comment]
Here is written that Economic Calendar is not implemented: https://pypi.org/project/investiny/
